oqs-template/ssh-keysign.c/open_keyfiles.fragment

```diff
@@ -0,0 +1,15 @@
+{%- for sig in config['sigs'] %}
+    {%- for variant in sig['variants'] %}
+	key_fd[i++] = open(_PATH_HOST_{{ variant['name'] }}_KEY_FILE, O_RDONLY);
+    {%- endfor %}
+{%- endfor %}
+	/* ADD_MORE_OQS_SIG_HERE */
```

Comment:
Could probably remove the `ADD_MORE_OQS_SIG_HERE` lines, since those were meant to be instructions to someone on where to look to add more algorithms from OQS, and now those are all done via the templating system.

Reply:
I did make a point to remove them, this one managed to slip through.